packages/circuits/config/index.ts

```diff
@@ -1,14 +1,29 @@
-import { SparseMerkleTree, hash4 } from '@unirep/utils'
+import { SparseMerkleTree, hash4, hash1 } from '@unirep/utils'
+import BN from 'bn.js'
 
 export const NUM_EPOCH_KEY_NONCE_PER_EPOCH = 3
 
-export const STATE_TREE_DEPTH = 9
+export const STATE_TREE_DEPTH = 12
 
-export const EPOCH_TREE_DEPTH = 9
-export const EPOCH_TREE_ARITY = 14
+export const EPOCH_TREE_DEPTH = 4
+export const EPOCH_TREE_ARITY = 3
 
-export const AGGREGATE_KEY_COUNT = 7
+export const SNARK_SCALAR_FIELD =
+    '21888242871839275222246405745257275088548364400416034343698204186575808495617'
```

Comment:
there is one `SNARK_FIELD_SIZE` in `@unirep/utils`
and it is exported from `maci-crypto`
I think we should not write it again

Reply:
I think we should not use the `SNARK_FIELD_SIZE` from `maci-crypto`. It's possible that maci (or us) switch to a different curve and the constant needs to be changed.